jsamchineme/send-it
# Fix order cancel button misbehaviour

#### What does this PR do?
- Fix order cancel button misbehaviour
#### Description of Task to be completed?
- Fix event listener errors
- Fix order cancel button misbehaviour
#### How should this be manually tested?
- pull branch and run _npm run start:dev_
- navigate to view order page after login, then click cancel order. This now works as expected
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/162889927
